### ISSUE TITLE
[GitHub] Search keywords have to be matched exactly in the issue title

### ISSUE THREAD

**Issue by StevenLi-DS** [raycast/extensions#5913]
### Extension

https://www.raycast.com/raycast/github

### Description

![cs-q8qpjyt1_2x](https://user-images.githubusercontent.com/33796896/231565177-7ef1e168-c931-487b-96a6-768b5a8d402f.png)
![cs-pn6rnflv_2x](https://user-images.githubusercontent.com/33796896/231565179-021d51f1-0537-4e47-a08e-c1030a052faf.png)

**Comment by thomaslombart** [comment]
Unfortunately, this seems to be a bug on GitHub's side... If you search this term within your issues, you shouldn't get any results as well. Could you try it?

**Comment by StevenLi-DS** [comment]
@thomaslombart Yes you are right man.. you need at least to type "undefined" to find the issue.
